tools/md2swagger.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/python
+
+# Convert lightblue metadata to swagger yaml
+#
+# This is useful in creating the first data model
+# for swagger specifications from lightblue metadata.
+# Run the script as:
+#
+#  md2swagger.py <metadata files> > model.yaml
+#
+#
+
+import sys
+import json
+import yaml
+
+def getTypeProperties(type):
+    properties=dict()
+    if type=='bigdecimal' or type=='biginteger':
+        properties['type']='string'
+    elif type=='boolean':
+        properties['type']='boolean'
+    elif type=='date':
+        properties['type']='string'
+        properties['format']='date-time'
+    elif type=='double':
+        properties['type']='number'
+        properties['format']='double'
+    elif type=='integer':
+        properties['type']='integer'
+        properties['format']='int64'
+    elif type=='string':
+        properties['type']='string'
+    return properties;
+
+def capital1(s):
+    return s[0].upper()+s[1:] 
+
+def singular(s):
+    if s[-3:]=='ies':
+        return s[0:-3]+'y'
+    elif s[-4:]=='sses':
+        return s[0:-4]+'ss'
+    elif s[-1]=='s':
+        return s[0:-1]
+    else:
+        return s;
```

Comment:
> Do you have a better idea to convert a plural to a singular?

In fairness, no. My thought was that a word that correctly ends with 's' would have problems, but the more I think about it, that may be a fringe case anyway.

Could you explain the need for this conversion?